Sources/Interaction/Style/InteractorStyleManipulator/index.js

```diff
@@ -504,11 +555,26 @@ function vtkInteractorStyleManipulator(publicAPI, model) {
   publicAPI.handleMouseMove = (callData) => {
     model.cachedMousePosition = callData.position;
     if (model.currentManipulator && model.currentManipulator.onMouseMove) {
+      const renderer = model.getRenderer(callData);
+      const beforeM = getCameraMatrix(renderer, tempCameraMatrix);
+
       model.currentManipulator.onMouseMove(
         model._interactor,
-        model.getRenderer(callData),
+        renderer,
         callData.position
       );
+
+      const newCenter = computeNewCenterOfRotation(
+        beforeM,
+        renderer,
+        model.centerOfRotation,
+        tempComputeObjects
+      );
+
+      if (!vec3.equals(model.centerOfRotation, newCenter)) {
```

Comment:
why bother doing a comparison check ? it will be done by the setter automatically, no ?

Reply:
Yes, setArray macro should take care of it.  fixed.